docs/pages/application-access/okta/guide.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: Setting up the Okta service
+description: How to set up the Okta service with some basic configurations.
+---
+
+This guide will help you set up the Okta service and provide some basic
+configurations to enable it.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
+
+- An Okta organization with an [Okta API token](https://developer.okta.com/docs/guides/create-an-api-token) provisioned.
+- A running Teleport cluster.
+- An instance where you will run the Okta service. This can live anywhere
+  with outbound access to Okta and must be running Linux.
+
+## Step 1/X. Create Okta import rules.
+
+Before setting up the Okta service, we should create Okta import rules in order
+to ensure that our Okta applications are set up with appropriate labeling before
+we attempt to synchronize them. First, we'll need to get the group and application
+IDs from Okta. The easiest way to do this is to use the <Var name="okta-api-token" />
+provisioned earlier.
+
+To get group IDs:
+
+```
+$ curl -H -H "Authorization: SSWS <Var name="okta-api-token" />" <Var name="okta-endpoint-url" />/api/v1/groups | jq '[.[] |  {"id": .id, "label": .profile.name}]'
+...
+[
+  {
+    "id": "111111",
+    "label": "Developers"
+  },
+  {
+    "id": "222222",
+    "label": "Admins"
+  },
+  {
+    "id": "333333",
+    "label": "Everyone"
+  },
+  {
+    "id": "444444",
+    "label": "HR"
+  },
+  ...
+]
+```
+
+```
+$ curl -H "Authorization: SSWS <Var name="okta-api-token" />" <Var name="okta-endpoint-url" />/api/v1/apps | jq '[.[] |  {"id": .id, "label": .label}]'
+...
+[
+  {
+    "id": "111111",
+    "label": "Okta Admin Console"
+  },
+  {
+    "id": "222222",
+    "label": "Okta Browser Plugin"
+  },
+  {
+    "id": "333333",
+    "label": "Okta Dashboard"
+  },
+  {
+    "id": "444444",
+    "label": "SAML test"
+  },
+  ...
+]
+```
+
+From here, you can copy the IDs from the output of the `curl` commands and use
+these to create the Okta import rule. Put this in a file called `okta-import-rule.yaml`:
+
+```yaml
+kind: okta_import_rule
+version: v1
+metadata:
+  name: import-rule
+  description: "Okta import rule for all groups and applications"
+spec:
+  mappings:
+  - match:
+    - app_ids: ["111111", "222222", "333333"] # Replace these with your Okta application IDs.
+    add_labels:
+      app_label: app-label-value
+  - match:
+    - group_ids: ["111111", "222222", "333333"] # Replace these with your Okta group IDs.
+    add_labels:
+      group_label: group-label-value
+```
+
+This will ensure that the applications and groups, once synchronized to Teleport,
+will be given the specified labels.
+
+## Step 2/3. Set up the Okta service.
+
+In this step, we'll set up the Okta service.
```

Comment:
I'd briefly explain here what Okta service does and why it's needed. Mention that it's the service that holds Okta API key and interacts with Okta API to import app/groups and update memberships, and that it connects back to Teleport cluster over reverse tunnel.